src/quo/components/avatars/account_avatar/view.cljs

```diff
@@ -17,18 +18,26 @@
     :customization-color - keyword or hexstring -> :blue/:army/... or #ABCEDF
    
     :theme - keyword -> :light/:dark"
-  [{:keys [size emoji]
-    :or   {size  style/default-size
-           emoji "🍑"}
+  [{:keys [size emoji account-name-initials emoji?]
+    :or   {size   style/default-size
+           emoji  "🍑"
+           emoji? true}
     :as   opts}]
   (let [theme      (quo.theme/use-theme)
         emoji-size (style/get-emoji-size size)]
     [rn/view
      {:accessible          true
       :accessibility-label :account-avatar
       :style               (style/root-container opts theme)}
-     [rn/text
-      {:accessibility-label      :account-emoji
-       :adjusts-font-size-to-fit true
-       :style                    {:font-size emoji-size}}
-      (when emoji (string/trim emoji))]]))
+     (if emoji?
+       [rn/text
+        {:accessibility-label      :account-emoji
+         :adjusts-font-size-to-fit true
+         :style                    {:font-size emoji-size}}
+        (when emoji (string/trim emoji))]
+       [text/text
+        {:accessibility-label :account-name
+         :size                :heading-1
+         :weight              :medium
+         :style               (style/account-name opts)}
+        account-name-initials])]))
```

Comment:
tiny suggestion 🔖 
maybe the `:accessibility-label` should be `:account-initials` or `:account-name-initials`?